### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Jaifroid**
I've updated the test instructions. Please check they're accurate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,18 +63,18 @@ sudo apt install zlib1g-dev libicu-dev libxapian-dev liblzma-dev
...
+* mounting a test archive in each of the four libzim builds;
 * checking the reported article count;
 * loading an article;
 * searching.
 
 Tests are run in Chromium browser context (needed in order to test WORKERFS) rather than purely in Node, so they are based on automation of the
 prototype, and are available in `/tests/prototype`.
 
-To run tests manually, replace the two `libzim-wasm.*.*` files in `tests/prototype` with the versions you wish to test (this is done automatically
+To run tests manually, replace the six `libzim-[w]asm.*.*` files in `tests/prototype` with the versions you wish to test (this is done automatically
```
[the remark this answers is not in the document]
Comment (Jaifroid) [reply README.md:77]: Yes, but that's how it was before -- if you build using the makefile, it copies the new builds to the test folder and tests with the new builds. I agree that's the way to do it. But this describes how to test files that you might have build elsewhere and that you want to test using the prototype. It explains that if you use the makefile then the copying will be done automatically. Is that OK?

[the remark this answers is not in the document]
Comment (Jaifroid) [reply README.md:77]: `(this is done automatically if you build using the provided Makefile)`

[the remark this answers is not in the document]
Comment (Rishabhg71) [reply README.md:77]: ahh my bad then I didn't consider someone who just wants to test without building. I will make these changes right away

[the remark this answers is not in the document]
Comment (Jaifroid) [reply README.md:77]: OK, would you be able to update the README, as I'm not sure I'll get it right? You can update it right here in GitHub with no fuss, just by going to the Files changed tab, click the three dots top right, and choose edit file.

**Pull request merged**